OKTA-413125 Overall style edit for OIE embedded auth content - part 1 (#2571)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/aspnet/redirecturi.md

>   If you're using the sample app, use `https://localhost:44314/interactioncode/callback`.

Reviewer: ```suggestion
  If you're using the sample app, use either `https://localhost:44314/interactioncode/callback` or
```

>   Otherwise, use a URI that your app accepts.

Reviewer: ```suggestion
a URI that is appropriate for your app.
```
